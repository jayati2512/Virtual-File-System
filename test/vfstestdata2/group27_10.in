mountvfs test10
createvfs /test10 100
createvfs test10 -199
createvfs test10 100
unmountvfs test10
mountvfs test10
unmountvfs test10
makedir / etc
mountvfs test10
makedir / tmp
makedir / boot
makedir / usr
makedir / etc/
makedir /etc/ jack
makedir /etc/animals/ dog
makedir /etc/animals/ horse
makedir /etc/animals/human/ jack
makedir /etc/animals/human/ bob
addfile /tmp body.h /usr/include/abc.h/abc
addfile /tmp legs.h /usr/include/wait.h
exportfile /tmp/body.h /tmp/tmppEL2Rt
exportfile /tmp/legs.h /tmp/tmppEL2Rt.h
copyfile /tmp/body.h /tmp/legs.h
addfile /tmp/body.h /usr/include/abc.h
deletedir /tmp/
deletedir /boot/
movedir /usr/ /etc/animals
listdir /etc/ 1 /tmp/etcdirs1.txt
listdir /etc/ 0 /tmp/etcdirs0.txt
searchfile /etc/animals/human/jack /tmp/alphabetagama.txt
searchfile jack /tmp/pulse-5j8vlM7MVoYe/pid
listfile /tmp/legs.h /tmp/listlegs.h
listfile /tmp/body.txt /tmp/listbody.txt
unmountvfs test10
