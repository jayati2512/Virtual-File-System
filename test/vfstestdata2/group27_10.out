mountvfs_FAILURE DATA_FILE_NOT_FOUND
createvfs_FAILURE INVALID_CHARACTER_IN_NAME
createvfs_FAILURE INVALID_SIZE
createvfs_SUCCESS
unmountvfs_FAILURE VFS_NOT_MOUNTED
mountvfs_SUCCESS
unmountvfs_SUCCESS
makedir_FAILURE VFS_NOT_MOUNTED
mountvfs_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_FAILURE INVALID_CHARACTER_IN_DIRNAME
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
makedir_SUCCESS
addfile_FAILURE UNABLE_TO_CREATE_NEWFILE
addfile_SUCCESS
exportfile_FAILURE CANNOT_FIND_SOURCEFILE
exportfile_SUCCESS
copyfile_FAILURE CANNOT_FIND_DESTINATIONPATH
addfile_FAILURE VFS_INSUFFICIENT_ARGUMENTS
deletedir_FAILURE DIRECTORY_IS_NOT_EMPTY
deletedir_SUCCESS
movedir_SUCCESS
listdir_SUCCESS
listdir_SUCCESS
searchfile_FAILURE CANNOT_FIND_SPECIFIED_OUTPUTFILE
searchfile_SUCCESS
listfile_FAILURE NOT_A_TEXT_FILE
listfile_FAILURE SOURCE_FILE_PATH_NOT_FOUND
unmountvfs_SUCCESS
